ims.yaml

```diff
@@ -0,0 +1,698 @@
+            format: date-time
+          formattedName:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 1
+            items:
+              $ref: '#/definitions/FormattedNameDType'
+          name:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 1
+            items:
+              $ref: '#/definitions/NameDType'
+          address:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/AddressDType'
+          contactInfo:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/ContactInfoDType'
+          demographics:
+            $ref: '#/definitions/DemographicsDType'
+          role:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/RoleDType'
+          agent:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/AgentDType'
+          credentials:
+            description: ...tbd (ComplexType property description)...
+            type: array
+            minItems: 0
+            items:
+              $ref: '#/definitions/CredentialsDType'
+        additionalProperties: false
+  PersonPatchBody:
+    type: object
+    required:
+      - data
+    additionalProperties: false
+    properties:
+      data:
+        type: object
+        required:
+          - type
+          - id
+          - attributes
+        properties:
+          type: *PersonType
+          id: *PersonId
+          attributes:
+            type: object
+            properties: *PersonProperties
+  PersonPostBody:
+    type: object
+    required:
+      - data
+    additionalProperties: false
+    properties:
+      data:
+        type: object
+        required:
+          - type
+          - attributes
+        properties:
+          type: *PersonType
+          attributes:
+            type: object
+            required:
+              - status
+              - dateLastModified
+              - formattedName
+              - name
+            properties: *PersonProperties
+  PersonResult:
+    description: ...tbd (Class description)...
+    type: object
+    properties:
+      data:
+        $ref: '#/definitions/PersonResource'
+    additionalProperties: false
+  PersonSetResult:
+    description: ...tbd (Class description)...
+    type: object
+    properties:
+      data:
+        type: array
+        minItems: 0
+        items:
+          $ref: '#/definitions/PersonResource'
+    additionalProperties: false
+  RoleDType:
```

Comment:
Since this definition is only used by `PersonResource`, could you move it somewhere before that definition?